### PR TITLE
Linter html css

### PR DESCRIPTION
**Pull request by PinkMoon25**
Created a feature branch "linter-Html-Css" and Set up a linter for HTML and CSS, I also added a .gitignore file and used the README.md template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html & Css
+
 
 ## Live Demo (if available)
```
Comment (ishimwezachee) [inline README.md:13]: - [ ] You've managed to use the Microverse template to document your project, but again the information you miss some information from your README.md file. some of the descriptions that you are missing are like, the project description, replace the dummy social media link, with your own social media links.   

Just replace Readme file information with you information corresponding to your project
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @PinkMoon25, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 



## Required Changes ♻️

- [ ] Pull Request should have a summary describing introduced changes in more detail.

_Check the comments under the review. for further request_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

File: README.md (hunk 1 above)
Comment (PinkMoon25) [reply README.md]: Thank You @ishimwe I have made the changes as requested, I don't have a Twitter handle so I deleted the Twitter link.

Comment (ishimwezachee) [reply README.md]: Its fine, you can add just the social media that you have

**Review by ishimwezachee — APPROVED** [review]
Hi @PinkMoon25, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

## Optional suggestions

- [OPTIONAL]  For there is no Testing or other configurations in your project, you can delete these so that your readme file can look more professional 
image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (PinkMoon25) [reply README.md]: Thank You for your help

**Pull request merged**